packages/auth/package.json

```diff
@@ -66,6 +68,7 @@
     "jest-openapi": "^0.14.2",
     "nock": "^13.5.4",
     "node-mocks-http": "^1.14.1",
+    "nodemon": "^3.1.0",
```

Comment:
```suggestion
```

Reply:
good catch, changed locally and updated the lockfile